src/main/java/fr/techad/edc/popover/model/HelpConfiguration.java

```diff
@@ -153,6 +155,18 @@
      */
     boolean isInternalBrowser();
 
+    /**
+     *
+     * @return Enum
+     */
+    HelpViewer getHelpViewer();
+
+    /**
+     *
+     * @param viewer
+     */
+    void setHelpViewer(HelpViewer viewer);
+
     /**
```

Comment:
Don't forget to remove the setInternalBrowser function. It is replaced by the HelpViewer